### ISSUE TITLE
Feature Request: Make the theme changing button into a dropdown select

### ISSUE THREAD

**Issue by penguinmoose** [theforumhelpers/theforumhelpers.github.io#335]
I think it would be better to turn the theme changing button into a dropdown menu/select. That way, someone can view all the available themes and jump to a desired one without having to cycle through all of them. Also, this is the usual way of choosing a theme - I thought that the change theme button actually opens a dropdown.

Also, it isn’t hard to style a dropdown. I’m pretty sure there are CSS selectors that activate with a certain selected item - that can even replace the CSS variables (I think).

**Comment by Accio1** [comment]
Standard dropdown elements (`<select>` tags) are actually a pain to style and end up looking like garbage if you view them on a device other than the one the styling was developed on. More likely if this were to be implemented, a custom dropdown that does not use the `<select>` element similar to the one currently used in the header on mobile devices.

**Comment by penguinmoose** [comment]
@Accio1 Right. I mean styling the dropdown opener (you can just style it like any other element).

Each browser has a different select option style, so I think we can forget about styling the interior (the options) of the select dropdown. The opener can be easily styled (at least from my experience; I might have forgotten).

Also, it might be worth checking out https://www.geeksforgeeks.org/how-to-style-a-dropdown-using-css/, although it is quite a bit of CSS.

A custom select like this one also requires tons of code (it also needs quite a bit of JS, though we can minify that?): https://www.w3schools.com/howto/howto_custom_select.asp

A lot of styling in my opinion is unneeded (we just have to set the font size, width, height, and maybe a few more things for it to be ok in my opinion). For now, can we not worry too much about the styling?

**Comment by penguinmoose** [comment]
@Accio1 Also, I think this needs some discussion (maybe add the tag?).
